src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -266,6 +266,18 @@ public net.minecraft.world.level.storage.LevelResource <init>(Ljava/lang/String;
 private-f net.minecraft.world.level.storage.loot.LootPool rolls # rolls
 private-f net.minecraft.world.level.storage.loot.LootPool bonusRolls # bonusRolls
 public net.minecraft.server.network.ServerConfigurationPacketListenerImpl finishCurrentTask(Lnet/minecraft/server/network/ConfigurationTask$Type;)V
+public net.minecraft.world.entity.npc.VillagerTrades$DyedArmorForEmeralds
```

Comment:
This would be better as a generated AT. See `projects/neoforge/build.gradle` for examples.